services/gateway-service/build.gradle

```
@@ -37,6 +37,7 @@ ext {
 dependencies {
 	implementation 'org.springframework.cloud:spring-cloud-starter-gateway'
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation 'org.apache.skywalking:apm-toolkit-micrometer-registry:8.9.0'
```

```suggestion
	implementation 'org.apache.skywalking:apm-toolkit-micrometer-registry:9.1.0'
```

I think 9.1 is a better for showcase(I know there is no actual change)

---

Also, I see the `songs-service` is using `org.apache.skywalking:apm-toolkit-logback-1.x:8.9.0`, should I need to update this to `9.1.0`?

---

Yes, let's bump them up together.